### PR TITLE
Added parsing for PAM SLA metrics

### PR DESCRIPTION
**Pull request by izzyblues**

### CODE REVIEW

File: extract.go [hunk 1]
```diff
@@ -20,11 +20,15 @@ var (
...
 
 	//[splunkMetrics] 2015/12/21 10:01:37.336610 UUID=08d30fb4-a7b3-11e5-955c-1e1d6de94879 readEnv=prod-uk transaction_id=tid_28pbiavoqs publishDate=1450692093737000000 publishOk=true duration=6 endpoint=content
 	pamRegex = regexp.MustCompile(`UUID=([\da-f-]*) readEnv=([\w-]*) transaction_id=(tid_[a-z0-9]*) publishDate=(\d*) publishOk=(\w*) duration=(\d*) endpoint=([\w-]*)`)
 
+	//[slaMetrics] 2015/12/21 10:01:37.336610 UUID=08d30fb4-a7b3-11e5-955c-1e1d6de94879 metPublishSla=true okEnvironments=[prod-uk,prod-us] transaction_id=tid_28pbiavoqs publishDate=1450692093737000000
+	//[slaMetrics] 2015/12/21 10:01:37.336610 UUID=08d30fb4-a7b3-11e5-955c-1e1d6de94879 metPublishSla=false okEnvironments=[prod-uk] transaction_id=tid_28pbiavoqs publishDate=1450692093737000000
+	slaPamRegex = regexp.MustCompile(`UUID=([\da-f-]*) metPublishSla=(\w*) okEnvironments=(\[[\w,\-]*\]) transaction_id=(tid_[a-z0-9]*) publishDate=(\d*)`)
```
Comment (peteclark-ft) [inline extract.go]: The field in PAM is `metPublishSLA` - which makes more sense to me (since SLA is an acronym)

File: extract.go [hunk 2]
```diff
@@ -203,6 +221,14 @@ type pamEntity struct {
 	Endpoint      string `json:"endpoint"`
 }
 
+type slaPamEntity struct {
+	UUID           string `json:"uuid"`
+	MetPublishSla  string `json:"metPublishSla"`
```
Comment (peteclark-ft) [inline extract.go]: Same as above, possibly should be `metPublishSLA`

**Comment by peteclark-ft** [comment]
Other than those two changes LGTM

**Comment by tosan88** [comment]
Is this still valid anymore?

**Comment by khatton-ft** [comment]
Closing this PR as it's so old. It's from a story we began but then abandoned, and it's unlikely we will take this approach going forward.

**Pull request closed without merging**